functions/Copy-DbaResourceGovernor.ps1

```diff
@@ -89,13 +89,17 @@ function Copy-DbaResourceGovernor {
         $sourceServer = Connect-SqlInstance -SqlInstance $Source -SqlCredential $SourceSqlCredential
         $destServer = Connect-SqlInstance -SqlInstance $Destination -SqlCredential $DestinationSqlCredential
 
-        $source = $sourceServer.DomainInstanceName
-        $destination = $destServer.DomainInstanceName
-
         if ($sourceServer.VersionMajor -lt 10 -or $destServer.VersionMajor -lt 10) {
             Stop-Function -Message "Resource Governor is only supported in SQL Server 2008 and above. Quitting."
```

Comment:
This can actually be removed if you want. Update the `Connect-SqlInstance` calls to include `-MinimumVersion 10`. This handles validating the version of each instance more gracefully.

Reply:
done, pushing